index.html

```diff
@@ -41,7 +41,12 @@
                 return (c === 'x' ? r : (r & 0x3 | 0x8)).toString(16);
             });
         }
+
         window.addEventListener('DOMContentLoaded', function () {
+            if (window.location.hostname !== 'redditp.com') {

```

Comment:
this exclusion should work correctly in all browsers, i suppose there is no need to send logs from dev or deploy preview environments

Reply:
`window.location.hostname === "www.redditp.com"` when I check.

```suggestion
            if (window.location.hostname.indexOf('redditp.com') === -1) {
```